package.json

```diff
@@ -15,7 +15,7 @@
     "coverage": "jest --coverage"
   },
   "dependencies": {
-    "@nypl/design-system-react-components": "3.1.3",
+    "@nypl/design-system-react-components": "3.1.4",
```

Comment:
This is helpful for removing the borders in the `List` component through a prop rather than having to add CSS to remove it.